tests/by-util/test_pgrep.rs

```diff
@@ -369,6 +369,34 @@ fn test_invalid_signal() {
         .stderr_contains("Unknown signal 'foo'");
 }
 
+#[test]
+#[cfg(target_os = "linux")]
+fn test_signals_that_never_match() {
+    new_ucmd!()
+        .arg("--require-handler")
+        .arg("--signal=0")
+        .arg(".*")
+        .fails()
+        .no_output();
+
+    new_ucmd!()
+        .arg("--require-handler")
+        .arg("--signal=KILL")
+        .arg(".*")
+        .fails()
+        .no_output();
+}
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_only_require_handler() {
+    new_ucmd!()
+        .arg("--require-handler")
+        .arg("--signal=0")
```

Comment:
What's the reason for this argument? Isn't this test about using `--require-handler` alone?

Reply:
It is there to make the exit status consistently 0, regardless of what else is running on the system. I guess alternative would be to ignore the exit status but just check that stderr doesn't contain the usage error

Reply:
I assumed it's the role of `--inverse` to ensure the exit status is consistently 0.

Reply:
Changed